wrhs-next-spec.yml

```diff
@@ -0,0 +1,230 @@
+openapi: 3.0.2
```

Comment:
Should this be `.gitignore`'d and generated on `prepare`?

Reply:
I'll have to make that happen with a separate PR (see comment in the title of this PR).